Main/src/Assertions/Code.cs

```diff
@@ -170,6 +170,25 @@ public static partial class Code
 		}
 		#endregion
 
+		#region Argument validation - valid count
+		/// <summary>Asserts if the passed value is not a valid count</summary>
```

Comment:
There should be a dot ('.') at the end of `<summary/>` content. This way documentation looks better.